test_conformance/events/test_userevents.cpp

```diff
@@ -29,6 +29,18 @@
 // CL error checking.
 
 #if defined(_MSC_VER)
+#if defined(__INTEL_LLVM_COMPILER)
```

Comment:
There is a single `CL_EXIT_ERROR` macro in "test_bufferreadwriterect.cpp" that doesn't require any ifdefs.  Can we use it here instead?  Even better, could we move it to a shared location and use it in both files?

Reply:
Thanks for pointing this out.
Yes, it's not necessary to define multiple `CL_EXIT_ERROR` and use ifdefs here. I have removed them.
I don't move `CL_EXIT_ERROR` to a shared location. `CL_EXIT_ERROR` in this file returns -1 for a non-CL_SUCCESS code, while `CL_EXIT_ERROR` in "test_bufferreadwriterect.cpp" does not do the same.